### PR TITLE
VIZTM-385: VGL_EXCLUDE for non-display GPUs

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

**Comment by tribal-tec** [comment]
Needs https://github.com/BlueBrain/VirtualGL/pull/2 to work

**Comment by tribal-tec** [comment]
FYI @hernando

File: eq/server/config/resources.cpp [hunk 1]
```diff
@@ -312,7 +316,19 @@ bool Resources::discover( ServerPtr server, Config* config,
             name << "display";
         }
         else
+        {
             name << "GPU" << ++gpuCounter;
+#ifdef EQUALIZER_USE_HWSD_gpu_glx
```
Comment (eile) [inline eq/server/config/resources.cpp]: Why is this GLX-specific? Maybe the detection is only implement in hwsd_glx, but afaik we do not need to restrict it here?

File: eq/server/config/resources.cpp [hunk 2]
```diff
@@ -312,7 +316,19 @@ bool Resources::discover( ServerPtr server, Config* config,
...
+            if( info.flags & hwsd::GPUInfo::FLAG_VIRTUALGL &&
+                info.device != LB_UNDEFINED_UINT32 )
+            {
+                std::ostringstream displayString;
+                displayString << info.nodeName << ":" << info.port << "."
+                              << info.device;
```
Comment (eile) [inline eq/server/config/resources.cpp]: Check with @hernando, iirc the nodeName should not be here.

**Comment by eile** [comment]
+1

File: eq/server/config/resources.cpp (hunk 2 above)
Comment (hernando) [inline eq/server/config/resources.cpp]: Shouldn't. VGL_EXCLUDE is only needed for local GPUs on the application node (which is were vglrun has been used) so there's non need to specify the host.

**Comment by hernando** [comment]
+2

**Pull request merged**